samples/XCT.Sample.Android/AccessiblePageRenderer.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Android.Content;
+using Xamarin.CommunityToolkit.Sample.Droid;
+using Xamarin.CommunityToolkit.Sample.Pages.Views;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+
+// This is a temporary fix for an issue in forms that will be fixed in a later release of 5.0
+// https://github.com/xamarin/Xamarin.Forms/pull/14089
+[assembly: ExportRenderer(typeof(SemanticOrderViewPage), typeof(AccessiblePageRenderer))]
+
+namespace Xamarin.CommunityToolkit.Sample.Droid
+{
+	public class AccessiblePageRenderer : PageRenderer
```

Comment:
On my VS (16.10.0. preview 2.1) this class isn't included in the project

![image](https://user-images.githubusercontent.com/20712372/117387850-fc4fb200-aebf-11eb-9a08-c0e5544ede13.png)

Maybe my VS is crazy because it is included on `csproj`,  can you double check that?